### ISSUE TITLE
Space before Emoticon is removed when selecting the emoticon from suggestion list

### ISSUE THREAD

**Issue by th-joerger** [mattermost/mattermost-mobile#1091]
#### Summary
Space before Emoticon is removed when selecting the emoticon from suggestion list

#### Environment Information

- Device Name: LG G4 (LG-H815)
- OS Version: 6.0 (3.10.84 kernel)
- Mattermost App Version: 1.3.0
- Mattermost Server Version: 4.1.0

#### Steps to reproduce
Type a text and try to insert emoticon :roll
------------------------------------------^here suggestions pop up. Select emoticon.

#### Expected behavior
Type a text and try to insert emoticon :roll_eyes:
```
Type a text and try to insert emoticon :roll_eyes:
```
(This renders properly in mobile app)

#### Observed behavior
Type a text and try to insert emoticon:roll_eyes:
```
Type a text and try to insert emoticon:roll_eyes:
```
(Space before emoticon removed. This doesn't render in mobile app, but renders in desktop app)

#### Possible fixes
No idea

**Comment by lindy65** [comment]
Hi @th-joerger,

Thank you for your report,

I have reproduced your issue and have created a [ticket](https://mattermost.atlassian.net/browse/RN-480) to track.

**Comment by th-joerger** [comment]
Thank you very much :)

**Comment by lfbrock** [comment]
This issue is fixed in v1.5.1
